src/ssh_audit/ssh_audit.py

```diff
@@ -947,19 +1002,21 @@ def process_commandline(out: OutputBuffer, args: List[str], usage_cb: Callable[.
         list_policies(out, aconf.verbose)
         sys.exit(exitcodes.GOOD)
 
-    if aconf.client_audit is False and aconf.target_file is None:
+    if aconf.client_audit is None and aconf.target_file is None:
         if oport is not None:
-            host = args[0]
+            host = host
         else:
-            host, port = Utils.parse_host_and_port(args[0])
-        if not host and aconf.target_file is None:
+            host = host
+            port = port
```

Comment:
`host = host` and `port = port` isn't needed.

Reply:
see above